### ISSUE TITLE
Pre-work Checklist: Developer: Heejung Hong

### ISSUE THREAD

**Issue by heejung-hong** [hackforla/website#5868]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] #6015
  - [x] #6066
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by heejung-hong** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

I'm not sure about the hours but it took me about three weeks to complete up to the first checkpoint.

**Comment by HackforLABot** [comment]
## Activity Log: heejung-hong
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- heejung-hong opened: https://github.com/hackforla/website/issues/5868 at 11/06/2023, 08:03 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/5868 at 11/06/2023, 08:03 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/5804 at 11/10/2023, 05:23 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5804#issuecomment-1806675791 at 11/10/2023, 08:35 PM PST
- heejung-hong opened: https://github.com/hackforla/website/pull/5898 at 11/13/2023, 09:33 AM PST
- heejung-hong PR was closed: https://github.com/hackforla/website/pull/5898#event-11017581386 at 11/20/2023, 03:03 PM PST
- heejung-hong opened: https://github.com/hackforla/website/pull/5925 at 11/20/2023, 03:24 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/5923#issuecomment-1823477643 at 11/22/2023, 12:41 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/5923#pullrequestreview-1745393467 at 11/22/2023, 01:18 PM PST
- heejung-hong PR was merged: https://github.com/hackforla/website/pull/5925#event-11041395757 at 11/22/2023, 01:30 PM PST
- heejung-hong issue closed by PR 5925: https://github.com/hackforla/website/issues/5804#event-11041395869 at 11/22/2023, 01:30 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/5798 at 11/22/2023, 04:31 PM PST
- heejung-hong opened: https://github.com/hackforla/website/pull/5937 at 11/27/2023, 10:13 AM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/5937#issuecomment-1831154211 at 11/28/2023, 07:26 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/5900#issuecomment-1831163830 at 11/28/2023, 07:36 PM PST
- heejung-hong closed: https://github.com/hackforla/website/pull/5937#event-11091458187 at 11/28/2023, 08:27 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/5937#issuecomment-1831197680 at 11/28/2023, 08:27 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/5900#issuecomment-1832626388 at 11/29/2023, 12:07 PM PST
- heejung-hong opened: https://github.com/hackforla/website/pull/5951 at 11/30/2023, 01:39 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/5952#pullrequestreview-1758942278 at 11/30/2023, 07:28 PM PST
- heejung-hong PR was merged: https://github.com/hackforla/website/pull/5951#event-11129560741 at 12/02/2023, 10:42 AM PST
- heejung-hong issue closed by PR 5951: https://github.com/hackforla/website/issues/5798#event-11129560787 at 12/02/2023, 10:42 AM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/5957#issuecomment-1846226938 at 12/07/2023, 02:48 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/5957#pullrequestreview-1771283121 at 12/07/2023, 03:19 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/5678 at 12/11/2023, 05:00 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5678#issuecomment-1867149393 at 12/21/2023, 06:21 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/5735 at 12/21/2023, 06:41 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5735#issuecomment-1867162344 at 12/21/2023, 06:44 PM PST
- heejung-hong issue closed as completed: https://github.com/hackforla/website/issues/5735#event-11376090693 at 01/02/2024, 10:56 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5678#issuecomment-1878112319 at 01/04/2024, 08:37 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6063#pullrequestreview-1805354133 at 01/04/2024, 09:09 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6061#pullrequestreview-1805371080 at 01/04/2024, 09:37 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5678#issuecomment-1882331309 at 01/08/2024, 07:27 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5678#issuecomment-1882336265 at 01/08/2024, 07:29 PM PST
- heejung-hong issue closed as completed: https://github.com/hackforla/website/issues/5678#event-11481903621 at 01/14/2024, 10:37 AM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5678#issuecomment-1893045818 at 01/15/2024, 08:16 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/5960 at 01/22/2024, 09:21 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5960#issuecomment-1905314650 at 01/22/2024, 09:24 PM PST
- heejung-hong opened: https://github.com/hackforla/website/pull/6149 at 01/23/2024, 08:37 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6164#pullrequestreview-1845135551 at 01/25/2024, 10:35 PM PST
- heejung-hong PR was merged: https://github.com/hackforla/website/pull/6149#event-11605512918 at 01/25/2024, 10:49 PM PST
- heejung-hong issue closed by PR 6149: https://github.com/hackforla/website/issues/5960#event-11605513020 at 01/25/2024, 10:49 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/4454 at 01/30/2024, 07:45 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/4454#issuecomment-1918331351 at 01/30/2024, 07:48 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6210#pullrequestreview-1858304708 at 02/01/2024, 08:59 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/6210#issuecomment-1922847957 at 02/01/2024, 09:18 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6207#pullrequestreview-1858325658 at 02/01/2024, 09:20 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6206#pullrequestreview-1858362360 at 02/01/2024, 09:41 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6204#pullrequestreview-1858377479 at 02/01/2024, 09:54 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6202#pullrequestreview-1858393836 at 02/01/2024, 10:09 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6201#pullrequestreview-1858430570 at 02/01/2024, 10:30 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6263#pullrequestreview-1871445502 at 02/08/2024, 03:41 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6241#pullrequestreview-1871459633 at 02/08/2024, 04:00 PM PST
- heejung-hong opened: https://github.com/hackforla/website/pull/6270 at 02/08/2024, 07:28 PM PST
- heejung-hong PR was merged: https://github.com/hackforla/website/pull/6270#event-11767719708 at 02/11/2024, 09:40 AM PST
- heejung-hong issue closed by PR 6270: https://github.com/hackforla/website/issues/4454#event-11767719759 at 02/11/2024, 09:40 AM PST
- heejung-hong commented on: https://github.com/hackforla/website/pull/6270#issuecomment-1937848629 at 02/11/2024, 11:38 AM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6301#pullrequestreview-1884247913 at 02/15/2024, 07:15 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6300#pullrequestreview-1884275343 at 02/15/2024, 08:03 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6357#pullrequestreview-1896844769 at 02/22/2024, 12:02 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6359#pullrequestreview-1896910802 at 02/22/2024, 12:44 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6358#pullrequestreview-1896923964 at 02/22/2024, 12:53 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6352#pullrequestreview-1897058184 at 02/22/2024, 02:18 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6351#pullrequestreview-1897092207 at 02/22/2024, 02:31 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6328#pullrequestreview-1897114725 at 02/22/2024, 02:51 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6394#pullrequestreview-1910049593 at 02/29/2024, 03:29 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6401#pullrequestreview-1910055866 at 02/29/2024, 03:36 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6386#pullrequestreview-1910098204 at 02/29/2024, 04:30 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6386#pullrequestreview-1912508783 at 03/01/2024, 06:25 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/5868#issuecomment-1974220678 at 03/01/2024, 06:34 PM PST
- heejung-hong closed as completed: https://github.com/hackforla/website/issues/5868#event-11986189379 at 03/01/2024, 06:35 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/6353 at 03/01/2024, 06:42 PM PST
- heejung-hong commented on: https://github.com/hackforla/website/issues/6353#issuecomment-1974228412 at 03/01/2024, 06:45 PM PST
- heejung-hong unassigned: https://github.com/hackforla/website/issues/6353 at 03/03/2024, 08:42 PM PST
- heejung-hong assigned: https://github.com/hackforla/website/issues/6353 at 03/03/2024, 08:44 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6438#pullrequestreview-1923879207 at 03/07/2024, 05:39 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6438#pullrequestreview-1925830907 at 03/08/2024, 01:00 PM PST
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6462#pullrequestreview-1937884719 at 03/14/2024, 04:24 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6461#pullrequestreview-1937890901 at 03/14/2024, 04:33 PM PDT
- heejung-hong commented on: https://github.com/hackforla/website/issues/6353#issuecomment-1999021728 at 03/14/2024, 11:28 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6508#pullrequestreview-1967772528 at 03/28/2024, 08:29 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6521#pullrequestreview-1967785050 at 03/28/2024, 08:41 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6533#pullrequestreview-1967790131 at 03/28/2024, 08:51 PM PDT
- heejung-hong commented on: https://github.com/hackforla/website/issues/6353#issuecomment-2028872754 at 03/31/2024, 11:52 AM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6538#pullrequestreview-1981464313 at 04/04/2024, 05:58 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6540#pullrequestreview-1981473019 at 04/04/2024, 06:09 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6553#pullrequestreview-1981496939 at 04/04/2024, 06:47 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6560#pullrequestreview-1981500887 at 04/04/2024, 06:53 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6564#pullrequestreview-1981592535 at 04/04/2024, 08:12 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6612#pullrequestreview-1995629057 at 04/11/2024, 04:27 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6616#pullrequestreview-1995636205 at 04/11/2024, 04:38 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6618#pullrequestreview-1995678913 at 04/11/2024, 05:40 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6624#pullrequestreview-1995777168 at 04/11/2024, 07:46 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6627#pullrequestreview-1995805926 at 04/11/2024, 08:29 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6629#pullrequestreview-1995809601 at 04/11/2024, 08:35 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6616#pullrequestreview-1995815901 at 04/11/2024, 08:45 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6626#pullrequestreview-1995831432 at 04/11/2024, 09:04 PM PDT
- heejung-hong submitted review: https://github.com/hackforla/website/pull/6626#pullrequestreview-2001923763 at 04/15/2024, 11:49 AM PDT
- heejung-hong commented on: https://github.com/hackforla/website/issues/6353#issuecomment-2065436514 at 04/18/2024, 03:35 PM PDT
- heejung-hong issue closed as completed: https://github.com/hackforla/website/issues/6353#event-12603175069 at 04/25/2024, 12:21 AM PDT
